<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SimpleCardWidget" name="SimpleCardWidget_2">
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="CardWidget" name="CardWidget_3">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="RadioButton" name="vgg_radio_btn">
             <property name="text">
              <string>vgg16</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="RadioButton" name="resent_radio_btn">
             <property name="text">
              <string>resent50</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="RadioButton" name="color_radio_btn">
             <property name="text">
              <string>颜色直方图</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="RadioButton" name="edge_texture_radio_btn">
             <property name="text">
              <string>边缘纹理</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="VerticalSeparator" name="VerticalSeparator"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="ToggleButton" name="auto_tbtn">
          <property name="text">
           <string>全自动模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ToggleButton" name="half_auto_tbtn">
          <property name="text">
           <string>半自动模式</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="VerticalSeparator" name="VerticalSeparator_3"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="SubtitleLabel" name="current_use_label">
          <property name="text">
           <string>当前使用的算法: </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="SubtitleLabel" name="Identification_label">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>识别相似度:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEdit" name="iden_label_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="VerticalSeparator" name="VerticalSeparator_4"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="PrimaryPushButton" name="load_model_btn">
            <property name="text">
             <string>加载模型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="impor_dataset_btn">
            <property name="text">
             <string>导入数据集</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="ToggleButton" name="start_predict_btn">
            <property name="text">
             <string>开启识别</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="SimpleCardWidget" name="SimpleCardWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="CardWidget" name="CardWidget_2">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="PixmapLabel" name="main_pic_pixmap"/>
             </item>
             <item>
              <widget class="PixmapLabel" name="subplot_pic_pixmap"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="CardWidget" name="CardWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="PrimaryPushButton" name="change_sup_pic_btn">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>切换副图</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PrimaryPushButton" name="different_class_btn">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>不相同类别</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PrimaryPushButton" name="same_class_btn">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>相同类别</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ListWidget" name="image_path_list">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToggleButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PixmapLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>VerticalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
